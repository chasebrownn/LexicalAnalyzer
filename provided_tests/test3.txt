t4 ((f)*).(f) #
" fff f ffffff "
