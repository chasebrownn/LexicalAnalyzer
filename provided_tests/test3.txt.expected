t4 , "fff"
t4 , "f"
t4 , "ffffff"
